package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ruff",
   "displayName": "Ruff",
   "description": "A Visual Studio Code extension with support for the Ruff linter.",
-  "version": "2024.17.0-dev",
+  "version": "2024.19.0-alpha",
```

Comment:
I think this still needs to be `-dev`.

Reply:
Possibly not, trying to read the `update_ext_version.py` code. I'd leave it as `-dev` though, it doesn't end up in any user-facing locations.